projects/slap_manipulation/configs/per_cmd_language.yaml

```diff
@@ -0,0 +1,27 @@
+"handover-to-person":
+  - "handover-to-person-approach-person"
+  - "handover-to-person-align-with-hand"
+  - "handover-to-person-release-object"
```

Comment:
What's the problem with hyphenation?

Reply:
i dont understand why, it's a question

Reply:
Just for consistency honestly. All strings we are using were hyphenated so this made sense. Is there a problem with hyphens being used for strings? Made it easier to write rather than adding punctuations.